t/unix.t

```diff
@@ -0,0 +1,142 @@
+use strict;
+use warnings;
+use Test::More tests=>45;
+use File::Temp qw(tempfile);
+use AnyEvent;
+use AnyEvent::Socket qw(tcp_server);
+use JSON qw(to_json from_json);
+our $CV;
+
+our $LAST_CLIENT;
+our $LAST_SERVER;
+our $t;
+
+our $TEST_MSG_COUNT=3;
+our $CONNECTION_COUNT=3;
+
+# lazy hack to force skip in unsupported platforms
+eval {
+  
+  my ($fh, $socket) = tempfile(UNLINK=>1);
+  $socket .=".sock";
+
+  use AnyEvent::WebSocket::Server;
+  use IO::Socket::UNIX;
+  {
+    my $server = IO::Socket::UNIX->new(
+      Type => SOCK_STREAM(),
+      Local => $socket,
+      Listen => 1,
+    ) or die "unsupported\n";
+  }
+};
+
+SKIP: {
+  skip 'unsupported',45 if $@;
```

Comment:
I would prefer to skip the entire test file rather than skip all 45 tests.  It is more obvious from the test summary that the test has been skipped.